packages/paste-website/src/component-examples/StatusPatternExamples.tsx

```diff
@@ -70,56 +165,76 @@ export const ConnectivityOffline = (): JSX.Element => {
-      color="colorTextIcon"	
-      decorative={false}	
-      title="draft" />	
-    <Text marginLeft="space20">Draft</Text>	
-  </Box>	
+<Box display="flex" rowGap="space100" flexWrap="wrap">	
+  <Box display="flex" columnGap="space50" rowGap="space60" flexWrap="wrap">
+    <Box display="flex">	
+      <ProcessErrorIcon	
+        color="colorTextIconError"	
+        decorative={false}	
+        title="error" />	
+      <Text marginLeft="space20">Error</Text>	
+    </Box>	
+    <Box display="flex">	
+      <ProcessWarningIcon	
+        color="colorTextIconWarning"	
+        decorative={false}	
+        title="warning" />	
+      <Text marginLeft="space20">Warning</Text>	
+    </Box>	
+    <Box display="flex">	
+      <ProcessSuccessIcon	
+        color="colorTextIconSuccess"	
+        decorative={false}	
+        title="success" />	
+      <Text marginLeft="space20">Success</Text>	
+    </Box>	
+    <Box display="flex">	
+      <ProcessNeutralIcon	
+        color="colorTextIconNeutral"	
+        decorative={false}	
+        title="neutral" />	
+      <Text marginLeft="space20">Neutral</Text>	
+    </Box>	
+    <Box display="flex">	
+      <ProcessInProgressIcon	
+        color="colorTextIconNeutral"	
+        decorative={false}	
+        title="In-progress" />	
+      <Text marginLeft="space20">In-progress</Text>	
+    </Box>	
+    <Box display="flex">	
+      <ProcessDisabledIcon	
+        color="colorTextIcon"	
+        decorative={false}	
+        title="disabled" />	
+      <Text marginLeft="space20">Disabled</Text>	
+    </Box>	
+    <Box display="flex">	
+      <ProcessDraftIcon	
+        color="colorTextIcon"	
+        decorative={false}	
+        title="draft" />	
+      <Text marginLeft="space20">Draft</Text>	
+    </Box>	
+  </Box>
+  <Box display="flex" columnGap="space50" rowGap="space60" flexWrap="wrap">
+    <StatusBadge variant="ProcessSuccess" size="borderless">Success</StatusBadge>	
+    <StatusBadge variant="ProcessError" size="borderless">Success</StatusBadge>	
+    <StatusBadge variant="ProcessWarning" size="borderless">Success</StatusBadge>	
+    <StatusBadge variant="ProcessNeutral" size="borderless">Success</StatusBadge>	
+    <StatusBadge variant="ProcessInProgress" size="borderless">Success</StatusBadge>	
+    <StatusBadge variant="ProcessDisabled" size="borderless">Success</StatusBadge>	
+    <StatusBadge variant="ProcessDraft" size="borderless">Success</StatusBadge>	
+  </Box>
+  <Box display="flex" columnGap="space50" rowGap="space60" flexWrap="wrap">
+    <StatusBadge variant="ProcessSuccess">Success</StatusBadge>	
+    <StatusBadge variant="ProcessError">Success</StatusBadge>	
+    <StatusBadge variant="ProcessWarning">Success</StatusBadge>	
+    <StatusBadge variant="ProcessNeutral">Success</StatusBadge>	
+    <StatusBadge variant="ProcessInProgress">Success</StatusBadge>	
+    <StatusBadge variant="ProcessDisabled">Success</StatusBadge>	
+    <StatusBadge variant="ProcessDraft">Success</StatusBadge>	
+  </Box>
```

Comment:
every status badge has 'Success' here, I think it should be based on their variant